src/main/java/tr/com/srdc/cda2fhir/transform/ICDATransformer.java

```diff
@@ -59,10 +62,26 @@
 	 *                        transformed
 	 * @param documentBody    The base64 decoded body from the original document
 	 * @param assemblerDevice An identifier with the name of the device doing the
-	 * 
+	 *
 	 * @return A FHIR Bundle that contains a Composition corresponding to the CDA
 	 *         document and all other resources that are referenced within the
 	 *         Composition.
 	 */
 	Bundle transformDocument(ContinuityOfCareDocument cda, String documentBody, Identifier assemblerDevice);
+
+	/**
+	 * Transforms a Clinical Document Architecture (CDA) instance to a Bundle of
+	 * corresponding FHIR resources
+	 *
+	 * @param cda             A ContinuityOfCareDocument (CDA) instance to be
+	 *                        transformed
+	 * @param documentBody    The base64 decoded body from the original document
+	 * @param assemblerDevice An identifier with the name of the device doing the
```

Comment:
Incomplete sentence for the assemblerDevice

Reply:
fixed.